tests/unit/r2i/tflite/engine.cc

```diff
@@ -68,12 +68,14 @@ TEST_GROUP (TfLiteEngine) {
   std::shared_ptr<r2i::IModel> model;
   std::shared_ptr<r2i::IModel> inc_model;
   std::shared_ptr<r2i::IFrame> frame;
+  int channels = 3;
```

Comment:
Can we get number of channels from frame information someway? Just trying to avoid fixed numbers